src/unifiedreport/agent/reportSummary.php

```diff
@@ -122,9 +122,15 @@ function summaryTable(Section $section, $uploadId, $userName, $mainLicenses, $li
 
     $table->addRow($rowWidth);
     $table->addCell($cellFirstLen, $cellRowContinue);
-    $table->addCell($cellSecondLen)->addText(htmlspecialchars(" Prepared by"), $firstRowStyle1, "pStyle");
+    $table->addCell($cellSecondLen)->addText(htmlspecialchars(" Report Created by"), $firstRowStyle1, "pStyle");
     $table->addCell($cellThirdLen)->addText(htmlspecialchars(" "
       .date("Y/m/d", $timestamp)."  ".$userName." (".$groupName.") "), null, "pStyle");
+
+    $table->addRow($rowWidth);
+    $table->addCell($cellFirstLen, $cellRowContinue);
+    $table->addCell($cellSecondLen)->addText(htmlspecialchars(" Analyzed by"),$firstRowStyle1, "pStyle");
+    $table->addCell($cellThirdLen)->addText(htmlspecialchars(" ".$assignedToUserName), null, "pStyle");
```

Comment:
Can the space in the beginning of the username be removed? It makes the alignment bit off and the value is already in a different table cell, it is not required actually.

And if you decide to remove it, can you please do the same for Created by field?